Documentation/Getting-Started/glossary.md

```diff
@@ -0,0 +1,135 @@
+# Glossary
+
+## Rook
+### Ceph CSI
+
+Ceph CSI plugins implement an interface between a CSI-enabled Container Orchestrator (CO) and Ceph clusters. For further information, please check out [CSI Driver documentation](../Storage-Configuration/Ceph-CSI/ceph-csi-drivers.md).
+
+### Krew Plugin
+
+The [Rook Krew plugin](../Troubleshooting/krew-plugin.md) is a tool to help troubleshoot your Rook cluster.
+
+### OpenShift
+
+[OpenShift](https://www.redhat.com/en/technologies/cloud-computing/openshift/container-platform) Container Platform is a distribution of the Kubernetes container platform.
+
+### Object Bucket Claim (OBC)
+
+An Object Bucket Claim (OBC) is custom resource which requests a bucket (new or existing) from a Ceph object store. For further reference please refer to [OBC Custom Resource](../Storage-Configuration/Object-Storage-RGW/ceph-object-bucket-claim.md).
+
+### Object Bucket (OB)
+
+An Object Bucket (OB) is a custom resource automatically generated when a bucket is provisioned. It is a global resource, typically not visible to non-admin users, and contains information specific to the bucket.
+
+### Toolbox
+
+The [Rook toolbox](../Troubleshooting/ceph-toolbox.md) is a container with common tools used for rook debugging and testing.
+
+## Rook CRDs
```

Comment:
How about removing this title, and just alphabetize all the CRDs as part of the main Rook section? The CRDs will anyway be naturally grouped when alphabetical.
```suggestion
```